cdap-security/src/main/java/co/cask/cdap/security/server/ExternalAuthenticationServer.java

```diff
@@ -230,10 +240,22 @@ protected void initHandlers() throws Exception {
   protected Executor executor() {
     final AtomicInteger id = new AtomicInteger();
     //noinspection NullableProblems
+    final Thread.UncaughtExceptionHandler h = new Thread.UncaughtExceptionHandler() {
+      @Override
+      public void uncaughtException(Thread t, Throwable e) {
+        if (Throwables.getRootCause(e) instanceof ServiceBindFailure) {
+          LOG.error(e.getMessage());
```

Comment:
Ah, the comment in StandaloneMain is confusing. I need to add "ExternalAuthenticationServer#execute()" to it.